packages/opentelemetry-exporter-collector/src/platform/node/util.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * It will remove http or https from the link as grpc requires link without
+ * protocol
+ * @param url
+ */
+export function fixGRPCUrl(url: string): string {
```

Comment:
a better name for this may be `removeProtocol` and the implementation can be simplified to:

```typescript
return url.replace(/^https?:\/\//, '');
```